regression/cbmc/xml-trace/test.desc

```diff
@@ -0,0 +1,25 @@
+CORE
+main.c
+--xml-ui --function test
+^EXIT=10$
+^SIGNAL=0$
+VERIFICATION FAILED
+<assignment assignment_type="state" base_name="u" display_name="test::u" hidden="false" identifier="test::u" mode="C" step_nr="\d+" thread="0">
+<location file=".*" function="test" line="\d+" working-directory=".*"/>
+<type>union myunion</type>
+<full_lhs>byte_extract_little_endian\(u, 0l, .*int.*\)</full_lhs>
```

Comment:
Please also add `--little-endian` to the command line to make sure this test is portable.